### PR TITLE
Bug 1444561: Establish a page with the benefits of FxAccounts

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Add /firefox/accounts/features page and associated assets.
Add A/B test for headlines.

## L10n

English only launch followed by an A/B test and some qualitative testing. The copy will be updated after the results of those tests. Then translation should be opened up.

## Issue / Bugzilla link

[Bug 1444561 - [Retention] - establish a page with benefits of FxAccounts detailed](https://bugzilla.mozilla.org/show_bug.cgi?id=1444561)

## Testing

- Different break points
- All images should have high res versions
- a/b test

### CODE REVIEW

**Comment by stephaniehobson** [comment]
I have #l10n questions I will ask about in the bug.

**Comment by stephaniehobson** [comment]
Still waiting for final sign off from the client but with one round of feedback in I'm not anticipating anything big. Code review can start :)

**Comment by stephaniehobson** [comment]
Small change coming. Please begin review though!

**Comment by alexgibson** [comment]
I got cc'd on an FxA bug and this comment seems relevant to this new page: https://bugzilla.mozilla.org/show_bug.cgi?id=1457004#c1

**Comment by alexgibson** [comment]
Not a blocker as this is for en-US only (right?), but the RTL layout could use some work (probably save for a future bug?).

https://screenshots.firefox.com/x3tOhE0gdTfmb3Ei/127.0.0.1

**Comment by alexgibson** [comment]
This has probably already been discussed, but the only other comment I'd add re: the design & content is that the images contain a lot of English text, which isn't going to be easy to translate?

File: media/css/pebbles/components/_global-nav.scss [hunk 1]
```diff
@@ -616,6 +616,7 @@ html.moz-nav-open .moz-global-nav-page-mask {
 }
 
 // Default (non-animated) drawer for old browsers.
+html, // for webkit
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/pebbles/components/_global-nav.scss]: Aw crap. My overflow: hidden doesn't work in Safari without it :(

[the remark this answers is not in the document]
Comment (alexgibson) [reply media/css/pebbles/components/_global-nav.scss]: If you need to hide overflow content in the page could you set it on `#outer-wrapper` instead?

[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/pebbles/components/_global-nav.scss]: It wasn't working when I set it on the sections but I'll try moving it up 👍

**Comment by stephaniehobson** [comment]
I'm going to save the RTL until the page is opened to #l10n.

The images have been discussed and no one has said to stop. Peiying also noted that pancakes are a regional food item. We basically have no way to translate them. The designers said they would generate localized images if the text was provided to them but we don't really have a way to get that to them. It's been brought up in Slack and on the Bug but do you think I should ping someone directly for an okay? With David away I guess that'd be Ben?

**Comment by stephaniehobson** [comment]
We could make the images SVGs on the page or text with some fancy CSS to re-create the browser chrome... now I want to play.

**Comment by stephaniehobson** [comment]
Updated to address all feedback!

Do I (theoretically) have control over which switches are enabled or do I have to ask someone with more permissions to toggle it?

**Comment by alexgibson** [comment]
> The images have been discussed and no one has said to stop. Peiying also noted that pancakes are a regional food item. We basically have no way to translate them. The designers said they would generate localized images if the text was provided to them but we don't really have a way to get that to them. It's been brought up in Slack and on the Bug but do you think I should ping someone directly for an okay? With David away I guess that'd be Ben?

I think as it's already been discussed and this is initially for en-US only there's no need to block on this. This is probably something our designers need to fix, so probably worht filing a bug for post-launch?

**Comment by alexgibson** [comment]
> Do I (theoretically) have control over which switches are enabled or do I have to ask someone with more permissions to toggle it?

Yep you should have control. Once this merges and is pushed to production, you'll need to file a PR against [this repo](https://github.com/mozmeao/www-config) where you can flip the switches for both stage and prod. Once that PR merges you can enable the config changes by running a script.

The full docs are here: https://mozmeao.github.io/www-config/

If you have any questions let me know!

**Pull request merged**